lib/cudadrv/stream.jl

```diff
@@ -11,6 +11,8 @@ Create a CUDA stream.
 """
 mutable struct CuStream
     handle::CUstream
+    valid::Bool
```

Comment:
Should this be `@atomic`? And the handle `const`

Reply:
The validity is only set from a finalizer, does that warrant making the field atomic?

Reply:
Technically possible to have more than one finalizer attached to an object and eventually possible that they might race each other, but currently no.

Reply:
But still a good suggestion, I'll add it.